src/utils/index.js

```diff
@@ -11,37 +11,29 @@ governing permissions and limitations under the License.
 */
 
 // Please keep in alphabetical order.
-export {
-  default as areThirdPartyCookiesSupportedByDefault
-} from "./areThirdPartyCookiesSupportedByDefault";
+export { default as areThirdPartyCookiesSupportedByDefault } from "./areThirdPartyCookiesSupportedByDefault";
```

Comment:
The rest of these changes come by virtue of having upgraded eslint. Sometime we should upgrade the eslint plugins we're using, but I don't want to cause more conflicts while there are open PRs.